cmd/eksctl-anywhere/cmd/upgradepackages.go

```diff
@@ -46,20 +46,5 @@ func upgradePackages(ctx context.Context) error {
 		return fmt.Errorf("unable to initialize executables: %v", err)
 	}
 
-	b := curatedpackages.NewBundleReader(
-		kubeConfig,
-		ipo.source,
-		deps.Kubectl,
-		nil,
-		nil,
-	)
-	activeController, err := b.GetActiveController(ctx)
-	if err != nil {
-		return err
-	}
-	err = b.UpgradeBundle(ctx, activeController, upo.bundleVersion)
-	if err != nil {
-		return err
-	}
-	return nil
+	return curatedpackages.UpgradePackages(ctx, kubeConfig, deps.Kubectl, string(upo.source), upo.bundleVersion)
```

Comment:
Is string conversion required here? The field is already a string.

Reply:
It's required if no modification is made to bundlesource. I can add it to this PR.